tools/c7n_azure/tests/cassettes/StorageUtilsTest.test_get_blob_client_from_storage_account_with_sas.yaml

```diff
@@ -0,0 +1,187 @@
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      User-Agent: [python/3.6.5 (Darwin-18.2.0-x86_64-i386-64bit) msrest/0.6.2 msrest_azure/0.6.0
+          azure-mgmt-storage/3.1.1 Azure-SDK-For-Python]
+      accept-language: [en-US]
+    method: GET
+    uri: https://management.azure.com/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/test_storage/providers/Microsoft.Storage/storageAccounts/cctstoragey6akyqpagdt3o?api-version=2018-07-01
+  response:
+    body:
+      string: !!binary |
+        H4sIAAAAAAAEAO29B2AcSZYlJi9tynt/SvVK1+B0oQiAYBMk2JBAEOzBiM3mkuwdaUcjKasqgcpl
+        VmVdZhZAzO2dvPfee++999577733ujudTif33/8/XGZkAWz2zkrayZ4hgKrIHz9+fB8/In7xR83b
+        9UePfvFHy2yRf/Too9dttpxl9ez3f/7q9Uejj9oir71PP/olo4/eFssZf1TV2UVObQr8ebdZT5pp
+        Xazaolo2d7Ps4cHDB/uT7fuzvWx7f/bwYDt7cJBt3zvYO7/3YPf+bOdBfrfOm2pdT/PP62q9au62
+        edP+/o2Avbuqq8tiltfN3S+KaV011Xk71i7vapvj6bRaL9vm7nTa6kfXn2Zvr3/RKruYtfcqQk0H
+        NdygvV6hwY19UNOymmYYHDUntNv5NF+2dVau8V2bXTRERKIO4b3KayIb/v5ombdXVf32eFryn5Pr
+        VdbQbx8d/2Bd56/z+rKYUsPRR5dF3a6z8oU0f7Uu6dNH3/s+0Xbl/THLz7N12R5PFY3jsqyuMCXN
+        erWq6rb5dtuumjd1dn5eTL9cltcfPTrPyiYffZQvp/U1Tw3QaEzH9Pt5URIBfjG1yCZlTjPZ1mt6
+        ocya9lQ+elMwDfd2dh9u7+xu7zx4s3vw6P7+o/ufju8//HR/f2f3p4DDpKwm3wAggvQ2v37NfEGN
+        exODrojGl0VDgymWF8SZLRq+Xk+neT6jjkcfTeucZ2pThzt7nz7cefhT1HpVF4usvj5dzlZVgamm
+        QchgPpqDno/uDjHYGM3G06rOx1ckFNVVM6YJv0swf9E6X6PrmwBwuyiEFiS7BQRuF4UgM3sjADSL
+        vM9kZsI838D3DVF/3Xx5/lKaUpPsMitK4PTRL/kl/w8OZP5zXAQAAA==
+    headers:
+      Cache-Control: [no-cache]
+      Content-Encoding: [gzip]
+      Content-Type: [application/json]
+      Date: ['Mon, 07 Jan 2019 22:04:43 GMT']
+      Expires: ['-1']
+      Pragma: [no-cache]
+      Server: ['Microsoft-Azure-Storage-Resource-Provider/1.0,Microsoft-HTTPAPI/2.0
+          Microsoft-HTTPAPI/2.0']
+      Strict-Transport-Security: [max-age=31536000; includeSubDomains]
+      Vary: [Accept-Encoding]
+      X-Content-Type-Options: [nosniff]
+      x-ms-correlation-request-id: [e9706541-b22f-47cd-a9c7-f6f0c05829e5]
+      x-ms-ratelimit-remaining-subscription-reads: ['11999']
+      x-ms-request-id: [498b104b-49ac-402c-b264-0679e037610f]
+      x-ms-routing-request-id: ['WESTUS2:20190107T220444Z:e9706541-b22f-47cd-a9c7-f6f0c05829e5']
+    status: {code: 200, message: OK}
+- request:
+    body: null
+    headers:
+      Accept: [application/json]
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      Content-Length: ['0']
+      User-Agent: [python/3.6.5 (Darwin-18.2.0-x86_64-i386-64bit) msrest/0.6.2 msrest_azure/0.6.0
+          azure-mgmt-storage/3.1.1 Azure-SDK-For-Python]
+      accept-language: [en-US]
+    method: POST
+    uri: https://management.azure.com/subscriptions/ea42f556-5106-4743-99b0-c129bfa71a47/resourceGroups/test_storage/providers/Microsoft.Storage/storageAccounts/cctstoragey6akyqpagdt3o/listKeys?api-version=2018-07-01
```

Comment:
Leaked keys here I think.

Reply:
Fine to leave this just make sure to delete that storage account or toss its keys.